DeAceroDLabs/marketplace-frontend
# [DMP-789] Remove products from shopping cart

## What is done

[[DMP-789] Vaciar carrito al crear orden](https://deacero.atlassian.net/browse/DMP-789)

## How is done

1. Refactor to Payment component.
  - Created a new PaymentPay component where the "pay" card lives along with the actions related to it.
  - Created a new OrderStatusPopup component to display and handle the actions once the order is sent and a response is received.
2. Added actions to the popup button to delete cart products from local storage and shopping cart context when an order is successfully created.

## How to test

Before running the app, please ping me so I can give you the credentials to test this feature successfully (apparently order service only works with my account). 

1. Run the project via `npm start`
3. log in to the app.
4. Add products to your cart.
5. Check your cart.
6. Move on to select an offer.
7. Move on to the Payment view.
8. Fill the card data with the following:
  - Card: 4111 1111 1111 1111
  - Expiry Date: 02/26
  - CVV: 444
9. Click on pay
10. Wait for the popup to show up.
